src/browser/api/file_download.ts

```diff
@@ -55,20 +60,24 @@ export function createWillDownloadEventListener(identity: Identity): (event: any
 
         try {
             const fileUuid: string = app.generateGUID();
-            const baseFileEvent: FileEvent = {
-                fileUuid,
+            const getFileEventData = (type: string, state: DownloadState): FileEvent => ({
```

Comment:
it's required for lambdas that return an object because the parser gets it confused with a statement block:

```
const someFunc = () => {  // a brace after a fat arrow is interpreted as a block
  return {
    prop: 'val'
  }
};
```

```
const someFunc = () => ({ // put it inside parenthesis and viola! it's an object
  prop: 'val'
});
```

Reply:
👍